src/Tizen.Multimedia.Vision/MediaVision/FacialLandmarkDetector.cs

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Collections;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using InteropInference = Interop.MediaVision.Inference;
+
+namespace Tizen.Multimedia.Vision
+{
+    /// <summary>
+    /// Provides the ability to detect facial landmarks on image source using inference engine.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public static class FacialLandmarkDetector
+    {
+        /// <summary>
+        /// Detects facial landmarks on the source image using inference engine set in <paramref name="config"/>.<br/>
```

Comment:
Suggestion:

        /// Detects facial landmarks in the image source using inference engine set in <paramref name="config"/>.<br/>